docs/source/deprecation.md

```diff
@@ -6,6 +6,13 @@ is evolving, it happens that some functionality needs to be changed, replaced
 or dropped completely. Such situations are inevitable. To reduce negative
 impact on your code, we introduce the deprecation process described below.
 
+## List of the deprecated functionality in leapp
+
+The following lists do not cover the functionality provided by particular leapp
```

Comment:
Not clear what "particular leapp repositories" means.

Reply:
Not sure how to write it. For example, `el7toel8` is a leapp repository. Soon we will add `el8toel9` leapp repository. These are the repositories containing leapp actors, leapp workflows, ... that leapp is executing. (in case of the official repositories, we are speaking about leapp repositories inside the *leapp-repository* git repository (https://github.com/oamg/leapp-repository).

Reply:
Miriam is right here, it is very confusing (even for me). Maybe instead of saying what the lists don't cover we should fix what is actually covered in this section (like framework stuff being deprecated etc)? This won't solve the general "leapp repositories" confusion, but will at least eliminate it here.

Reply:
Synced with Miriam in person. The paragraph is now reworded @fernflower